scenes/npcs/talker/talker.gd

```diff
@@ -20,11 +22,17 @@ func _ready() -> void:
 	DialogueManager.dialogue_ended.connect(_on_dialogue_ended)
 
 
-func _on_interaction_started() -> void:
+func _on_interaction_started(from_right: bool) -> void:
+	_previous_look_at_side = look_at_side
+	if from_right and look_at_side != NPC.LookAtSide.RIGHT:
+		look_at_side = NPC.LookAtSide.RIGHT
+	elif not from_right and look_at_side != NPC.LookAtSide.LEFT:
+		look_at_side = NPC.LookAtSide.LEFT
 	DialogueManager.show_dialogue_balloon(dialogue, "", [self])
 
 
 func _on_dialogue_ended(_dialogue_resource: DialogueResource) -> void:
+	look_at_side = _previous_look_at_side
```

Comment:
I wonder if it would feel more natural without this – if you talk to them from the right, have them turn to face you, and stay looking in that direction after the conversation since there is no reason for them to turn back around?